vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -208,6 +209,13 @@ private VCFHeader addMissingMetadataLines(VCFHeader header) {
         if (header.getFormatHeaderLine("GT") == null) {
             header.addMetaDataLine(new VCFFormatHeaderLine("GT", 1, VCFHeaderLineType.String, "Genotype"));
         }
+        // CSQ line
+        if (header.getFormatHeaderLine("CSQ") == null) {
+            header.addMetaDataLine(new VCFInfoHeaderLine("CSQ", 1, VCFHeaderLineType.String,
```

Comment:
Yes. it would be good also instead of using the Java Properties class, use a custom one, with methods for the properties we are using (like getVariantCollection(), includeAnnotation() ...) 

Spring boot also has support for this:
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties